assets/panel/debugger.properties

```diff
@@ -400,6 +402,7 @@ editor.conditionalPanel.close=Cancel edit breakpoint and close
 # LOCALIZATION NOTE (editor.jumpToMappedLocation1): Context menu item
 # for navigating to a source mapped location
 editor.jumpToMappedLocation1=Jump to %S location
+editor.jumpToMappedLocation1.accesskey=C
```

Comment:
You need to pick a letter available in the associated label, otherwise this will show up as 

> Jump to %S location (C)

With C underlined.

I know you're just de-hardcoding the values, but it should be fixed nonetheless, unless you've completely run out of characters.